esx_service/cli/vmdkops_admin.py

```diff
@@ -1286,6 +1286,7 @@ def config_init(args):
         auth = auth_data.AuthorizationDataManager(db_path)
         err = auth.new_db()
         if err:
+            os.remove(db_path)
```

Comment:
Yes, I made a branch in the same source tree that had this change from #1259 and it pulled the changes from 1259 as well.